WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -38,14 +42,18 @@ import javax.inject.Inject
 import javax.inject.Named
 
 private const val SWITCH_TO_MY_SITE_DELAY = 500L
+private const val ONE_SITE = 1
 
 class WPMainActivityViewModel @Inject constructor(
     private val featureAnnouncementProvider: FeatureAnnouncementProvider,
     private val buildConfigWrapper: BuildConfigWrapper,
     private val appPrefsWrapper: AppPrefsWrapper,
     private val analyticsTracker: AnalyticsTrackerWrapper,
     private val quickStartRepository: QuickStartRepository,
-    @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher
+    @Named(UI_THREAD) private val mainDispatcher: CoroutineDispatcher,
+    private val mSelectedSiteRepository: SelectedSiteRepository,
+    private val mAccountStore: AccountStore,
+    private val mSiteStore: SiteStore,
```

Comment:
@SowmiaS 👍  on adding these dependencies to the `WPMainActivityViewModel` from the Activity. We can rename these variables without the 'm' prefix as that was a convention followed in the java code. for eg: `mSiteStore` can be renamed to `siteStore`. wdyt? 🤔

Reply:
I agree @AjeshRPai. Done in [commit](https://github.com/wordpress-mobile/WordPress-Android/pull/16063/commits/ec02f37cda30b9213a1bfb3c563fff4f0869a0c5).